build-farm/platform-specific-configurations/alpine-linux.sh

```diff
@@ -49,7 +49,7 @@ export GNUPGHOME
 
 BOOT_JDK_VARIABLE="JDK${JDK_BOOT_VERSION}_BOOT_DIR"
 if [ ! -d "$(eval echo "\$$BOOT_JDK_VARIABLE")" ]; then
-  bootDir="$PWD/jdk-$JDK_BOOT_VERSION"
```

Comment:
Just noting that for these versions it doesn't matter whether they have the `-` or not as it's a temporary directory used to download the build anyway, but no harm in it being consistent with the "default` top level path names on the machines that we will be creating.